tests/feature/parametrized.feature

```diff
@@ -2,3 +2,7 @@ Scenario: Parametrized given, when, thens
     Given there are <start> cucumbers
     When I eat <eat> cucumbers
     Then I should have <left> cucumbers
+
+
+Scenario: Parametrized given - single param
```

Comment:
what is this test asserting? There is no "then" step

Reply:
You already have asked that question :). Shall we add comment somewhere in the code explaining what this test is testing?

https://github.com/pytest-dev/pytest-bdd/pull/272#discussion_r236071403